src/+energy/assembleCoefficientMatrices.m

```diff
@@ -59,11 +59,12 @@
             - (C1(n - 1, 2) / Delt_t + C4_a(n - 1)) * SV.hh(n - 1) - (C1(n, 1) / Delt_t + C4(n, 1)) * SV.hh(n) ...
             + (C3(n - 1, 2) / Delt_t) * P_g(n - 1) + (C3(n, 1) / Delt_t) * P_g(n) ...
             + (C1(n - 1, 2) / Delt_t) * SV.h(n - 1) + (C1(n, 1) / Delt_t) * SV.h(n);
+
     elseif ~ModelSettings.Soilairefc && ModelSettings.Thmrlefc
         RHS(indxBotm) = -C7(indxBotm) + (C2(indxBotm, 1) * SV.T(indxBotm) + C2(indxBotm, 2) * SV.T(indxBotm + 1)) / Delt_t ...
             - (C1(indxBotm, 1) / Delt_t + C4(indxBotm, 1)) * SV.hh(indxBotm) - (C1(indxBotm, 2) / Delt_t + C4(indxBotm, 2)) * SV.hh(indxBotm + 1) ...
             + (C1(indxBotm, 1) / Delt_t) * SV.h(indxBotm) + (C1(indxBotm, 2) / Delt_t) * SV.h(indxBotm + 1);
-        for i = 2:ModelSettings.NL
+        for i = indxBotm:ModelSettings.NL
```

Comment:
Running the model fails due to this change. In this file, if `~GroundwaterSettings.GroundwaterCoupling` , then `indxBotm=1`, the equation` ARG4 = C1(i - 1, 2) / Delt_t;` returns an error because i-1 is 0. Can you fix it?

Reply:
> Running the model fails due to this change. In this file, if `~GroundwaterSettings.GroundwaterCoupling` , then `indxBotm=1`, the equation` ARG4 = C1(i - 1, 2) / Delt_t;` returns an error because i-1 is 0. Can you fix it?

My mistake, I correct it now